.github/workflows/lint.yml

```
@@ -7,11 +7,7 @@ jobs:
       - uses: actions/checkout@master
       - uses: errata-ai/vale-action@master
         with:
-          styles: |
-            https://github.com/errata-ai/Microsoft/releases/latest/download/Microsoft.zip
-            https://github.com/errata-ai/Google/releases/latest/download/Google.zip
-            https://github.com/errata-ai/write-good/releases/latest/download/write-good.zip
-          config: https://raw.githubusercontent.com/vaadin/docs/latest/.vale.ini
+          config: https://raw.githubusercontent.com/vaadin/docs/vale-config/.github/.vale-pr.ini
```

Update before merging:

```suggestion
          config: https://raw.githubusercontent.com/vaadin/docs/latest/.github/.vale-pr.ini
```